src/views/annual-report/2020/l10n.json

```diff
@@ -0,0 +1,379 @@
+
+    "annualReport.2020.communityTitle": "Community",
+    "annualReport.2020.communityIntro": "In 2020, the Scratch Community became an even more vital place for young people to find a sense of togetherness and belonging. As we saw the meaningful conversations, collaborative projects, and moving stories Scratchers shared, we were in awe of their creative and resilient spirit.",
+    
+    "annualReport.2020.communityTitle1": "Virtual Family Creative Coding Nights Guide",
+    "annualReport.2020.communityText1": "In 2019, with support from Google.org, the Scratch Team worked with Chicago Public School’s Office of Computer Science to connect students, families, teachers, and other community members through Family Creative Coding Nights.",
+    "annualReport.2020.communityText2": "This year, our teams faced a new challenge: how could we bring the playful, community-building spirit of Family Creative Coding Nights to a virtual space, helping schools develop vital connections with remote students and their families? We developed the Virtual Family Coding Nights guide to provide a structure for these connections and support joyful learning",
+    
+    "annualReport.2020.communityQuoteName": "Kendra Mallory, M.Ed.",
+    "annualReport.2020.communityQuoteTitle": "Ruggles Elementary S.T.E.M. Coordinator",
+    "annualReport.2020.communityQuoteText": "[In 2020], there were not many opportunities to engage with parents in such a fun, high energy way. So this opportunity provided much needed engagement...Teachers were apprehensive, but the students' level of excitement pushed them into a space where they had to trust the process and allow kids to learn from one another.",
+
+    "annualReport.2020.communityScratchCommunity": "Scratch Community",
+    "annualReport.2020.communityScratchCommunityIntro": "When asked why they use Scratch, most Scratchers talk about the importance of the online community for motivating their ongoing participation, providing a space where they can express their creativity, make friends, receive feedback, get new ideas, and learn new skills. Many Scratchers express their appreciation for the Scratch community as a safe and welcoming space to connect, share, and learn from one another.",
+
+    "annualReport.2020.communityQuoteGroupName1": "angelical",
+    "annualReport.2020.communityQuoteGroupText1": "I joined Scratch when I was 11 years old and the things I learned from using the platform and interacting with the community were really a vital part of my learning growing up.",
+    "annualReport.2020.communityQuoteGroupName2": "dlore2009",
+    "annualReport.2020.communityQuoteGroupText2": "Scratch has allowed me to do things from home, like\n- Respect people and their projects\n- Make friends\n- Feel that I am not alone in this quarantine\n....and much more, so I want to say ¡GRACIAS!",
```

Comment:
@chrisgarrity do the `\n`s in this string look okay or does it need to be broken up into separate l10n strings?